addons.conf

```diff
@@ -11,9 +11,13 @@ map $http_user_agent $mobile_agents {
 server {
     listen  80 default;
 
-    # This is required to thwart nginx's file
-    # cache when using virtual box.
-    sendfile off;
+    location /code/storage/files/ {
+        internal;
```

Comment:
Unsure if this is necessary but it certainly doesn't hurt.

I wonder, that kinda means that we won't be able to hit that URL ourselves in the browser, e.g for testing right?

Reply:
> Unsure if this is necessary but it certainly doesn't hurt.

That's what the Nginx page suggests but yeah, that will prevent us to hit that URL ourselves..